### PR TITLE
Arch based distros full system upgrade

### PR DESCRIPTION
**Pull request by SandupaTech**
fixed arch based linux distros full system upgrading while checking what distro is it.

### CODE REVIEW

File: install-release.sh [hunk 1]
```diff
@@ -203,7 +203,7 @@ identify_the_operating_system_and_architecture() {
       PACKAGE_MANAGEMENT_REMOVE='zypper remove'
       package_provide_tput='ncurses-utils'
     elif [[ "$(type -P pacman)" ]]; then
-      PACKAGE_MANAGEMENT_INSTALL='pacman -Syu --noconfirm'
+      PACKAGE_MANAGEMENT_INSTALL='pacman -Syy --noconfirm'
```
Comment (yuhan6665) [inline install-release.sh:206]: 不好意思 这个项目一直没人看pr
我看了一下似乎没有必要每次强制更新已有的包吧

**Comment by yuhan6665** [comment]
先关了

File: install-release.sh (hunk 1 above)
Comment (chise0713) [reply install-release.sh:206]: -Syu是更新所有包，-Syy是重载软件库缓存

Comment (yuhan6665) [reply install-release.sh:206]: 感谢两位 更新了 https://github.com/XTLS/Xray-install/commit/a69543ab82f284f561f923815112ab3a4eaab1f8

**Pull request closed without merging**